integration_test/warehouse/integration_test.go

```diff
@@ -1501,6 +1506,151 @@ func TestUploads(t *testing.T) {
 	})
 }
 
+func TestDestinationTransformation(t *testing.T) {
+	c := testcompose.New(t, compose.FilePaths([]string{"../../warehouse/integrations/testdata/docker-compose.transformer.yml"}))
```

Comment:
For consistency I would use rudder-go-kit for the transformer. 

https://github.com/rudderlabs/rudder-go-kit/blob/main/testhelper/docker/resource/transformer/transformer.go